profile/Repository names

```diff
@@ -0,0 +1,19 @@
+blog: Home to the Blog for Public Code and where blog posts are planned and collectively written.
+
+standard: The Standard for Public Code - a model for public organizations to build their own open source solutions to enable successful future reuse by similar public organizations in other places.
+
+jitsi-community: A tool that allows users to sign up to a Jitsi instance that is not open to everyone, using a web UI so that they can start sessions by themselves.
+
+illustrations: A collection of illustrations meant to be reused in the communications of the Foundation for Public Code.
+
+publiccode.net: The website of the Foundation for Public Code that is the most public facing and the primary place for people to learn about the organization.
+
+governance-game: Card game used to learn what needs of governance organizations collaborating on public code codebases has.
+
+projects: Home to a list of useful resources developed by the Foundation for Public Code.
```

Comment:
```suggestion
projects: Home to a list of useful projects developed by the Foundation for Public Code.
```
Perhaps I'm too About'ized, but I think of About as a place with resources. What is unique about projects? Time bound? In collaboration with other organizations? Anything we could add here as a qualifier?

Reply:
Relevant discussion is in https://github.com/publiccodenet/projects/issues/10 which I think makes a good case for referring to them as resources.